### ISSUE TITLE
Support for 2FA

### ISSUE THREAD

**Issue by mpxgit** [kontron/redmine_oauth#26]
Hello Kontron,

is the 2FA with Azure AD supported. 
We get the following error after login with username and password.

interaction_required: AADSTS50076: Due to a configuration change made by your administrator, or because you moved to a new location, you must use multi-factor authentication to access. 

Normaly the second authentication screen should appear.

**Comment by picman** [comment]
Yes, it is supported. Check your plugin's settings and your app registration. The plugin itself has nothing to do with 2FA. It just hands over the authentication to Azure and waits for its callback to be called.

**Comment by mpxgit** [comment]
We have found a solution to force using the MFA.
According to Microsoft [OAuth 2.0 Protocol Extensions](https://learn.microsoft.com/en-us/openspecs/windows_protocols/ms-oapx/0fc398ca-88d0-4118-ae60-c3033e396e60) we added amr_values=ngcmfa to the 'Azure AD' authorize_url.

`authorize_url: "/#{Setting.plugin_redmine_oauth[:tenant_id]}/oauth2/authorize?amr_values=ngcmfa",`

Since ruby is not my language i don't think that this ist the correct way.
